index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
check general layout requirements
1. Content: max-width: 1200px
2 The second sub-section area has a height of 100px.
...etc